far/dialog.cpp

```diff
@@ -2561,6 +2555,7 @@ bool Dialog::ProcessKey(const Manager::Key& Key)
 			case KEY_NUMENTER:
 			case KEY_ENTER:
 			case KEY_NUMPAD5:
+			case KEY_F5:
```

Comment:
Why F5?
The PR mentions Shift+F5, so seems to be unrelated to columns.

Reply:
None of the `case`s above include any modifiers. I originally thought (without looking anywhere) that `LocalKey()` somehow strips off the modifiers, but apparently it is not.

The code below seems to be about ensuring menu redraw when the selected item was changed by the key. I am not sure why it should be done from the dialog and not in the menu itself. If indeed the intent was to handle redraw on selection change, neither `KEY_NUMPAD5` nor `KEY_(SHIFT)F5` should be here because they cannot change the selection. And when I comment out these two `case`s, things _seem_ to work fine.

I did not dig deeper and changed `KEY_F5` to `KEY_SHIFTF5`. This place should probably be revisited, but not in this PR if possible. Please let me know.

Reply:
Looks like KEY_NUMPAD5 [was added by you](https://github.com/FarGroup/FarManager/commit/bf0d80386e38c19fb0000d2e97d991587c241797#diff-6e956c0db925004801c1509ab2884997c52d3bc9f5f44c9af810b204d6a08a55R2551) as well :) for purposes Ctrl+Numpad5.

The key is definitely not stripped of any modifiers here.
Since these keys alone have nothing to do with the intended actions and, as you say, things seem to work fine without them, may I suggest removing both KEY_NUMPAD5 and KEY_SHIFTF5 in this PR?
We probably have way too much dead code already.

Reply:
Done!